base/bucket_test.go

```diff
@@ -356,46 +361,62 @@ func mockCertificatesAndKeys(t *testing.T) {
 	clientKey, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
 	saveAsKeyFile(t, clientKeyPath, clientKey)
 	certBytes, err = x509.CreateCertificate(rand.Reader, &clientTemplate, &rootTemplate, &clientKey.PublicKey, rootKey)
-	assert.NoError(t, err, "Client certificate should be generated")
+	require.NoError(t, err, "Client certificate should be generated")
 	saveAsCertFile(t, clientCertPath, certBytes)
+
+	require.True(t, fileExists(rootKeyPath), "File %v should exists", rootKeyPath)
+	require.True(t, fileExists(rootCertPath), "File %v should exists", rootCertPath)
+	require.True(t, fileExists(clientKeyPath), "File %v should exists", clientKeyPath)
+	require.True(t, fileExists(clientCertPath), "File %v should exists", clientCertPath)
+
+	return
```

Comment:
Generally want to avoid naked returns

```suggestion
	return certPath, clientCertPath, clientKeyPath, rootCertPath, rootKeyPath
```

Reply:
Makes more sense, naked return has been avoided in latest revision.